src/components/20-molecules/top-content-bar/child.scss

```diff
@@ -1,5 +1,17 @@
-axa-top-content-bar > axa-link {
-  padding: 5px;
+axa-top-content-bar {
+  axa-link {
+    padding: 5px;
+  }
+
+  a {
```

Comment:
The text displayed in the bar is entered in page properties in rich text area where the user can also add the links. That link is not axa-link but the ordinary a[href] so I had to add some CSS here to make it look like the link in the design. This will not affect the axa-link-aem child components, if there are any.